### ISSUE TITLE
missing pickle port

### ISSUE THREAD

**Issue by stbuehler** [ahupowerdns/metronome#21]
carbon-relay sends data only over pickle protocol, metronome only accepts plaintext

**Comment by ahupowerdns** [comment]
can you clarify? is this "please add pickle protocol support"?

**Comment by stbuehler** [comment]
Yes; it would make it easier to run carbon-cache and metronome behind the same carbon sink, or distribute the same data to multiple metronome instances.

**Comment by ahupowerdns** [comment]
I can't find a specification of what is pickled in carbon pickle protocol,
nor which version of pickle is used.  I also can't find a specification
for pickle itself. So if anyone wants this, please dig up these details.

**Comment by stbuehler** [comment]
Now that I have graphite running I probably won't go back to metronome, so I'm not that much interested - I just wanted to leave feedback about the problems I was having.

Perhaps it would also be an option to access a whisper database of an already running carbon-cache.
